specs/Standards/MultiToken/Events.md

```diff
@@ -0,0 +1,155 @@
+  EVENT_JSON: {
+    standard: string,
+    version: string,
+    event: MtEvent,
+    data: MtMintLog[] | MtBurnLog[] | MtTransferLog[] | MtApprovalLog[]
+  }
+}
+
+// Minting event log. Emitted when a token is minted/created. 
+// Requirements
+// * Contract MUST emit event when minting a token
+// Fields 
+// * Contract token_ids and amounts MUST be the same length 
+// * `owner_id`: the account receiving the minted token
+// * `token_ids`: the tokens minted
+// * `amounts`: the number of tokens burned, wrapped in quotes and treated
+//    like a string, although the numbers will be stored as an unsigned integer
+// .  array with 128 bits.
+interface MtMintLog {
+    owner_id: string,
+    token_ids: string[],
+    amounts: string[],
+}
+
+// Burning event log. Emitted when a token is burned.  
+// Requirements
+// * Contract MUST emit event when minting a token
+// Fields 
+// * Contract token_ids and amounts MUST be the same length 
+// * `owner_id`: the account receiving the minted token
+// * `token_ids`: the tokens to burned
+// * `amounts`: the number of tokens burned, wrapped in quotes and treated
+//    like a string, although the numbers will be stored as an unsigned integer
+// .  array with 128 bits.
+interface MtBurnLog {
+    owner_id: string,
+    token_ids: string[]
+    amounts: string[]
+}
+
+// Transfer event log. Emitted when a token is transfered.  
+// Requirements
+// * Contract MUST emit event when transferring a token
+// Fields 
+// * `sender_id`: the account sending the minted tokens
+// * `receiver_id`: the account receving the minted tokens
+// * `token_ids`: the tokens to transfer 
+// * `amounts`: the number of tokens burned, wrapped in quotes and treated
+//    like a string, although the numbers will be stored as an unsigned integer
+// .  array with 128 bits.
+interface MtTransferLog {
+    sender_id: string,
+    receiver_id: string,
+    token_ids: string[],
+    amounts: string[]
+}
+
+// Transfer event log. Emitted when a token is transfered.  
+// Requirements
+// * Contract MUST emit event when transferring a token
+// * Contract token_ids and amounts MUST be the same length 
+// Fields 
+// * `sender_id`: the account sending the minted tokens
+// * `receiver_id`: the account receving the minted tokens
+// * `token_ids`: the tokens to transfer 
+// * `amounts`: the number of tokens burned, wrapped in quotes and treated
+//    like a string, although the numbers will be stored as an unsigned integer
+// .  array with 128 bits.
+interface MtTransferLog {
+    sender_id: string,
+    receiver_id: string,
+    token_ids: string[],
+    amounts: string[]
+}
+
+// Approval event log. Emitted when a token's approval has changed.  
+// Requirements
+// * Contract MUST emit event when approval of tokens have changed
+// * Contract token_ids and amounts MUST be the same length 
+// Fields 
+// * `account_id`: the account sending the approval 
+// * `approved_id`: the account being approved
```

Comment:
```suggestion
// * `approved_account_id`: the account being approved
```
Similarly, I saw this argument in the NFT standard and think it'll be good to be as consistent as possible